doc/tutorial/single_cell_model.rst

```diff
@@ -48,7 +48,7 @@ Our *single-segment HH cell* has a simple morphology and dynamics, constructed a
     labels = arbor.label_dict({'soma':   '(tag 1)',
                                'midpoint': '(location 0 0.5)'})
 
-    # (3) Create cell and set properties
+    # (3) Create and set up a decor object
```

Comment:
Can you make the same change to `single_cell_model.py`?

Reply:
Sure, thanks for pointing it out :)